xchange-ftx/src/main/java/org/knowm/xchange/ftx/service/FtxTradeService.java

```diff
@@ -8,20 +8,31 @@
 import org.knowm.xchange.dto.trade.MarketOrder;
 import org.knowm.xchange.dto.trade.OpenOrders;
 import org.knowm.xchange.dto.trade.UserTrades;
+import org.knowm.xchange.exceptions.ExchangeException;
 import org.knowm.xchange.ftx.FtxAdapters;
 import org.knowm.xchange.ftx.FtxAuthenticated;
+import org.knowm.xchange.ftx.dto.FtxResponse;
 import org.knowm.xchange.ftx.dto.account.FtxAccountDto;
+import org.knowm.xchange.ftx.dto.trade.FtxOrderDto;
+import org.knowm.xchange.instrument.Instrument;
 import org.knowm.xchange.service.trade.TradeService;
 import org.knowm.xchange.service.trade.params.CancelOrderParams;
+import org.knowm.xchange.service.trade.params.DefaultOrderHistoryParamsInstrumentSpan;
 import org.knowm.xchange.service.trade.params.DefaultTradeHistoryParamInstrument;
+import org.knowm.xchange.service.trade.params.InstrumentParam;
+import org.knowm.xchange.service.trade.params.OrderHistoryParams;
+import org.knowm.xchange.service.trade.params.OrderHistoryParamsTimeSpan;
 import org.knowm.xchange.service.trade.params.TradeHistoryParams;
 import org.knowm.xchange.service.trade.params.orders.DefaultOpenOrdersParamInstrument;
 import org.knowm.xchange.service.trade.params.orders.OpenOrdersParams;
+import org.knowm.xchange.utils.Assert;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 import java.io.IOException;
 import java.util.Collection;
+import java.util.Date;
+import java.util.List;
 
 public class FtxTradeService extends FtxTradeServiceRaw implements TradeService {
```

Comment:
Please make sure `FtxTradeService#createTradeHistoryParams` returns params object that implements `TradeHistoryParamsTimeSpan`, i.e. trade history can be queried within specified time frame.